tests/test_components.py

```diff
@@ -35,4 +35,4 @@ def test_parameter_variants():
 
     assert converted_list == expected
     assert list(converted_tuple) == expected
-    assert list(converted_set) == expected
+    assert list(converted_set).sort() == expected.sort()
```

Comment:
This is not doing what you believe is doing: you're comparing `None == None`, because `list.sort()` doesn't return anything:
```suggestion
    assert sorted(converted_set) == sorted(expected)
```

Reply:
The other option (convenient if `converted_set` is already a `set`) is just to use `set` operations:
```suggestion
    assert len(converted_set ^ set(expected)) == 0
```
https://docs.python.org/3/library/stdtypes.html#frozenset.symmetric_difference